Cargo.toml

```diff
@@ -25,7 +25,7 @@ chrono = "0.4"
 clap = "2"
 download = { path = "download" }
 error-chain = "0.12"
-effective-limits = "0.4"
+effective-limits = { git = "https://github.com/tblah/effective-limits.rs.git", rev = "5fc9ca2fcf7c698d585325d3aad4a8bbe20b4418" }
```

Comment:
I take it that there needs to be a patch merged and a new release made for this?